steps/check-your-appeal/CheckYourAppeal.js

```diff
@@ -1,20 +1,32 @@
 'use strict';
 
-const { Question, goTo } = require('@hmcts/one-per-page');
+const {
+    CheckYourAnswers: CYA,
+    section
+} = require('@hmcts/one-per-page/checkYourAnswers');
+
+const { goTo } = require('@hmcts/one-per-page');
 const paths = require('paths');
 
-class CheckYourAppeal extends Question {
+class CheckYourAppeal extends CYA {
 
     get url() {
+
         return paths.checkYourAppeal
     }
 
-    get form() {}
+    sections() {
+
+        return [
+            section('benefit-type', { title: 'Benefit' }),
+            section('mrn-date',     { title: 'Mandatory Reconsideration Notice (MRN)' }),
+            section('appointee',    { title: 'Appointee' })
+        ];
+    }
 
     next() {
         return goTo(this.journey.Confirmation);
```

Comment:
You'll want to change this once I have the statement of truth for check your answers in. Check your answers needs to be a "Stop" so that the walk tree knows that if you are editing a step before check your answers you should always stop and check the answers before continuing. A pause in the tree if you know what I mean.

Reply:
Still working on that functionality though so I'm not sure what the interface will be.

Reply:
Sounds good, look forward to it.